recipes/crispector/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$PYTHON setup.py install --single-version-externally-managed --record=rec.txt
```

Comment:
Please remove this build script and add this to the build section instead (avoids an additional file, also pip install is the recommended way to install Python packages):
```
script: python -m pip install . --no-deps -vv
```

Reply:
Thanks.